src/components/Home/ChangeLog.js

```diff
@@ -46,7 +54,13 @@ const ChangeLog = (props) => {
               <li>
                 <Typography>
                   <Typography>
-                    {t("2020/07/28：")}
+                    {t("{{date, YYYY/MM/DD}}：", {
```

Comment:
i18next formatting is seems to unnecessary if you use `Intl.DateTimeFormat`
```suggestion
                    {t("{{date}}：", {
```